FirebaseAuth/Tests/SampleSwift/AuthenticationExample/ViewControllers/SettingsViewController.swift

```diff
@@ -113,14 +113,19 @@ class SettingsViewController: UIViewController, DataSourceProviderDelegate {
   }
 
   private func toggleAccessGroup() {
-    if AppManager.shared.auth().userAccessGroup == nil {
-      guard let bundleDictionary = Bundle.main.infoDictionary,
-            let group = bundleDictionary["AppIdentifierPrefix"] as? String else {
-        fatalError("Configure AppIdentifierPrefix in the plist")
+    do {
+      if AppManager.shared.auth().userAccessGroup == nil {
+        guard let bundleDictionary = Bundle.main.infoDictionary,
+              let group = bundleDictionary["AppIdentifierPrefix"] as? String else {
+          fatalError("Configure AppIdentifierPrefix in the plist")
+        }
+        try AppManager.shared.auth().useUserAccessGroup(group +
+                                                        ".com.google.firebase.auth.keychainGroup1")
```

Comment:
The `AppIdentifierPrefix` added in https://github.com/firebase/firebase-ios-sdk/pull/13599 has a trailing "."
```suggestion
                                                        "com.google.firebase.auth.keychainGroup1")
```